### PR TITLE
Spike end to end test system

### PR DESCRIPTION
**Pull request by cbaines**
Branch for https://trello.com/c/0DFOQTnE/845-spike-gnu-guix-system-for-running-end-to-end-tests

Very half finished, please ask any questions, including those not specifically related to end to end testing.

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -0,0 +1,36 @@
+-*- mode: org -*-
```
Comment (philandstuff) [inline README]: if you name this file `README.org` then Github will render it 😄

Comment (cbaines) [reply README]: Great, I've now renamed it, and even added fancy babel stuff :)

**Comment by cbaines** [comment]
There is a Guix branch for GOV.UK puppet now available https://github.com/alphagov/govuk-puppet/tree/guix which will install Guix within the development vm.

**Comment by cbaines** [comment]
Merging as the spike is now complete.

**Pull request merged**